internal/configmodels/configmodels.go

```diff
@@ -69,6 +69,7 @@ type Exporters map[string]Exporter
 // Processor is the configuration of a processor. Specific processors must implement this
 // interface and will typically embed ProcessorSettings struct or a struct that extends it.
 type Processor interface {
+	NamedEntity
```

Comment:
Where is this used?

Reply:
SetName() is called in internal/configv2/configv2.go. The name is used in internal/collector/processor/nodebatcher/factory.go

Reply:
I see, thanks for clarifying.